tests/integration/scenario.go

```diff
@@ -40,6 +40,7 @@ func initScenario(ctx *godog.ScenarioContext) {
 	ctx.Step(`^Virtual service "([^"]*)" exposing service "([^"]*)" by gateway "([^"]*)" is configured in namespace "([^"]*)"$`, steps.CreateVirtualService)
 	ctx.Step(`^Request with header X-Forwarded-For with value "([^"]*)" sent to httpbin should return X-Envoy-External-Address with value "([^"]*)"$`, steps.ValidateHeader)
 	ctx.Step(`^Request with header "([^"]*)" with value "([^"]*)" sent to httpbin should return "([^"]*)" with value "([^"]*)"$`, steps.ValidateHeader)
+	ctx.Step(`^"([^"]*)" "([^"]*)" in namespace "([^"]*)" is applied`, steps.ResourceIsApplied)
```

Comment:
Applied might suggest that the resource is applied as part of the test run
```suggestion
	ctx.Step(`^"([^"]*)" "([^"]*)" in namespace "([^"]*)" is present`, steps.ResourceIsApplied)
```